addons/phone_validation/models/mail_thread_phone.py

```diff
@@ -43,6 +43,32 @@ class PhoneMixin(models.AbstractModel):
         help="Indicates if a blacklisted sanitized phone number is a mobile number. Helps distinguish which number is blacklisted \
             when there is both a mobile and phone field in a model.")
 
+    def search_phone_mobile_search(self, operator, value):
+
+        if len(value) <= 2:
+            raise UserError(_('Please enter at least 3 digits when searching on phone / mobile.'))
```

Comment:
return None

Reply:
mmh ?

Reply:
I think that returning None will cause an error. The search method has to return a search domain, therefore I think that if we want to return empty results, we should instead return a search domain containing a condition which is always equals to False.

Reply:
A runbot test is faling for an unkown reason when this line is changed, because of this and for now, I will leave the raise error statement unchanged.